### PR TITLE
feat: add support assets for Mantle chain

### PR DESCRIPTION
**Pull request by myz1237**
https://lifi.atlassian.net/browse/LF-8064

### CODE REVIEW

File: src/coins/coins.ts [hunk 1]
```diff
@@ -606,6 +606,10 @@ export const basicCoins: BasicCoin[] = [
         address: '0xf0F161fDA2712DB8b566946122a5af183995e2eD',
         decimals: 6,
       },
+      [ChainId.MNT]: {
```
Comment (myz1237) [inline src/coins/coins.ts:609]: USDT

File: src/coins/coins.ts [hunk 2]
```diff
@@ -877,6 +881,10 @@ export const basicCoins: BasicCoin[] = [
         address: '0xd988097fb8612cc24eeC14542bC03424c656005f',
         decimals: 6,
       },
+      [ChainId.MNT]: {
```
Comment (myz1237) [inline src/coins/coins.ts:884]: USDC

File: src/coins/coins.ts [hunk 3]
```diff
@@ -1106,6 +1114,10 @@ export const basicCoins: BasicCoin[] = [
         address: '0x3C1BCa5a656e69edCD0D4E36BEbb3FcDAcA60Cf1',
         decimals: 8,
       },
+      [ChainId.MNT]: {
```
Comment (myz1237) [inline src/coins/coins.ts:1117]: WBTC

File: src/coins/coins.ts [hunk 4]
```diff
@@ -1216,6 +1228,10 @@ export const basicCoins: BasicCoin[] = [
         address: '0x4200000000000000000000000000000000000006',
         decimals: 18,
       },
+      [ChainId.MNT]: {
```
Comment (myz1237) [inline src/coins/coins.ts:1231]: WETH

**Review by melianessa — APPROVED** [review]

**Pull request merged**